### PR TITLE
PWA for subdomains

### PR DESCRIPTION
**Pull request by vidya-ram**

### CODE REVIEW

File: hasjob/templates/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": {% if g.board %}"{{ g.board.title }}"{% else %}"Hasjob"{% endif %},
+  "short_name": {% if g.board %}"{{ g.board.title}}"{% else %}"hasjob"{% endif %},
```
Comment (jace) [inline hasjob/templates/manifest.json]: Default "hasjob" should be "Hasjob"

File: hasjob/templates/manifest.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (jace) [inline hasjob/templates/manifest.json]: Filename needs to be `manifest.json.jinja2`.

File: hasjob/views/index.py [hunk 3]
```diff
@@ -830,16 +830,19 @@ def search():
...
 @app.route('/service-worker.js', methods=['GET'])
 def sw():
     return app.send_static_file('service-worker.js')
 
 
+@app.route('/manifest.json', methods=['GET'], subdomain='<subdomain>')
 @app.route('/manifest.json', methods=['GET'])
 def manifest():
-    return app.send_static_file('manifest.json')
+    return render_template('manifest.json')
```
Comment (jace) [inline hasjob/views/index.py]:
```python
return Response(render_template('manifest.json.jinja2'), mimetype='application/json')
```

File: hasjob/templates/manifest.json (hunk 1 above)
Comment (vidya-ram) [reply hasjob/templates/manifest.json]: Fixed

File: hasjob/templates/manifest.json (hunk 2 above)
Comment (vidya-ram) [reply hasjob/templates/manifest.json]: Fixed

File: hasjob/views/index.py (hunk 3 above)
Comment (vidya-ram) [reply hasjob/views/index.py]: Fixed

**Pull request merged**